tests/resources/mlflow-test-plugin/setup.py

```diff
@@ -11,5 +11,6 @@
         "mlflow.tracking_store": "file-plugin=mlflow_test_plugin:PluginFileStore",
         "mlflow.artifact_repository":
             "file-plugin=mlflow_test_plugin:PluginLocalArtifactRepository",
+        "mlflow.context_provider": "unused=mlflow_test_plugin:PluginContextProvider"
```

Comment:
QQ: Although this is `unused`, does it have to be unique per plugin? That is, is it the key of some map?

Reply:
Not sure - will look at validating this.

Reply:
I made a quick copy of the test plugin called 'other' and looks like it's just appended to a list:

```
>>> import entrypoints
>>> entrypoints.get_group_all("mlflow.run_context_provider")
[EntryPoint('unused', 'other', 'PluginRunContextProvider', None), EntryPoint('unused', 'mlflow_test_plugin', 'PluginRunContextProvider', Distribution('mflow_test_plugin', '0.0.1')), EntryPoint('unused', 'other', 'PluginRunContextProvider', Distribution('other', '0.0.1'))]
```

The 'unused' string just becomes an attribute on these `EntryPoint` objects that we don't care about. Other packages that use this mechanism (e.g. `pytest_mock`) give this unused part a name that corresponds to the package, i.e.:

```
"mlflow.context_provider": "mlflow_test_plugin=mlflow_test_plugin:PluginContextProvider"
```

Happy to make this change if you'd like the setup.py of the test plugin to serve as an example to plugin developers.

Reply:
Got it. I guess as long as it works either way, seems fine not to have this as an example.